ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6734,3 +6736,10 @@ OAQM:
   - outside_air_co2_concentration_sensor
   implements:
   - MONITORING
+
+OAIDC:
```

Comment:
If you do not have visibility to these points, there is no need to model this at all.

Reply:
I removed them.